library/hawkular_alerts_group.py

```diff
@@ -0,0 +1,183 @@
+requirements: [ hawkular/hawkular-client-python ]
+author: Daniel Korn (@dkorn)
+options:
+  hawkular_api_hostname:
+    description:
+      - the hawkular API hostname
+    default: HAWKULAR_HOSTNAME env var if set, otherwise it is required to pass it
+    required: True
+  hawkular_api_port:
+    description:
+      - the hawkular API port
+    default: HAWKULAR_PORT env var if set, otherwise it is required to pass it
+    required: True
+  hawkular_api_auth_token:
+    description:
+      - the hawkular API auth token
+    default: HAWKULAR_TOKEN env var if set, otherwise it is required to pass it
+    required: True
+  tenant:
+    description:
+      - the hawkular tenant
+    default: null
+    required: True
+  name:
+    description:
+      - the group trigger name
+    default: null
+    required: False
+  group_id:
+    description:
+      - the group trigger id
+    default: null
+    required: True
+  severity:
+    description:
+      - the group trigger severity
+    default: null
+    required: False
+  scheme:
+    description:
+      - the hawkular scheme
+    default: 'https'
+    required: False
+    choices: ['https', 'http']
+  cafile:
+    description:
+      - the path to the ca file
+    default: null
+    required: False
+  state:
+    description:
+      - the state of the user
+      - On present, it will create the group trigger
+      if it does not exist
+    required: True
+    choices: ['present', 'absent']
+  enabled:
+    description:
+      - whether the group trigger should be enabled or not
+    required: False
+    default: True
+'''
+
+EXAMPLES = '''
+# Add example group trigger
+  hawkular_alerts_group:
+    hawkular_api_hostname: 'hawkular-endpoint.example.com'
+    port: 443
+    token: '******'
+    tenant: '_system'
+    group_id: 'example-group-trigger'
+    name: 'Example Group Trigger'
+    severity: 'high'
+    state: 'present'
+'''
+
+import urllib2
+import hawkular.alerts
+
+# TODO: remove once the ssl cert verification is checked
+import os
+
+
+class HawkularAlertsGroupTrigger(object):
+    """ Hawkular Alerts object to create group triggers in Hawkular
+    """
+    def __init__(self, module, tenant, hostname, port, scheme, token, context):
+        self.module  = module
+        self.client  = hawkular.alerts.HawkularAlertsClient(tenant, host=hostname, port=port, scheme=scheme, token=token, context=context)
+        self.changed = False
+
+    def create_group_trigger(self, name, group_id, severity, enabled):
+        """ Creates a group trigger in Hawkular Alerting component
+
+            Returns:
+                whether or not a change took place and a short message
+                describing the operation executed
+        """
+        try:
+            self.client.get_trigger(group_id)
+            return dict(
+                msg="Group trigger {group_id} already exist".format(group_id=group_id),
+                changed=self.changed)
```

Comment:
What if it exists but has different name, severity, etc.?
If you prefer to do it later, please open an issue.